aptos-move/block-executor/src/view.rs

```diff
@@ -1546,6 +1833,13 @@ mod test {
     #[derive(Clone, Debug)]
     struct TestTransactionType {}
 
+    // #[derive(Debug, PartialEq, Eq)]
```

Comment:
dead comment?
was this for testing? then please try to adjust / re-use types from existing tests in block executor and mvhashmap

Reply:
Removed it.